README.md

```diff
@@ -86,7 +71,7 @@ Use `/var/run/vde.bridged.en0.ctl` as the VDE socket path.
 ### Testing without launchd
 
 ```console
-make
+make install.vde-2
```

Comment:
Shouldn't this be `sudo make install.vde-2`?

And doesn't `make install.bin` require `sudo` as well?